x-pack/plugins/uptime/server/lib/requests/search/enrich_monitor_groups.ts

```diff
@@ -311,25 +315,23 @@ const getHistogramForMonitors = async (
         },
       },
       aggs: {
-        by_id: {
-          terms: {
-            field: 'monitor.id',
-            size: queryContext.size,
+        histogram: {
+          auto_date_histogram: {
+            field: '@timestamp',
+            // 12 seems to be a good size for performance given
+            // long monitor lists of up to 100 on the overview page
+            buckets: 12,
+            missing: 0,
           },
           aggs: {
-            histogram: {
-              auto_date_histogram: {
-                field: '@timestamp',
-                buckets: QUERY.DEFAULT_BUCKET_COUNT,
-                missing: 0,
+            by_id: {
+              terms: {
+                field: 'monitor.id',
+                size: monitorIds.length,
```

Comment:
Some of the tests are failing due to this change; when `monitorIds.length` is `0` ES will return an error and the request blows up.